interfaces/console/src/app/console/subscribers/page.tsx

```diff
@@ -190,7 +190,6 @@ const Page = () => {
       setIsSubscriberDetailsOpen(true);
       setSelectedSubscriber(id);
       setIsPackageActivationNeeded(shouldActivatePackage);
-
       getSubscriber({
         variables: {
           subscriberId: id,
```

Comment:
Why we're calling `getSubscriber` again on multiple occasions? We already have all subscribers info when we call `useGetSubscribersByNetworkQuery`. Please double check this implementation